spec/spec_helper.rb

```diff
@@ -74,4 +74,14 @@ def self.run_all(reporter = nil)
       example.run
     end
   end
+
+  # Work around ammeter's incompatibility with RSpec 4
+  # TODO: Remove once https://github.com/alexrothenberg/ammeter/pull/64 is merged
+  generator_path_regex = Regexp.compile(%w[spec generators].join('[\\\/]') + '[\\\/]')
+  config.include Ammeter::RSpec::Rails::GeneratorExampleGroup,
+                 type: :generator
+  config.include Ammeter::RSpec::Rails::GeneratorExampleGroup,
+                 file_path: lambda { |file_path, metadata|
+                   metadata[:type].nil? && generator_path_regex =~ file_path
+                 }
```

Comment:
Do we even need this? The issue in ammeter is the use of both tags and file, we presumedly have one of them if we're getting failures?

Reply:
Removed, works :+1: 
I'll squash the last commit.